catlearn/fingerprint/adsorbate_fingerprint.py

```diff
@@ -37,19 +41,35 @@
                      'ionenergies']
 
 
+# Text based feature.
+facetdict = {'001': [1.], '0001step': [2.], '100': [3.],
+             '110': [4.], '111': [5.], '211': [6.], '311': [7.],
+             '532': [8.]}
+
+
 def check_length(labels, result, atoms):
+    """Check that two lists have the same length. If not, print an informative
+    error message containing a databse id if present.
+    Parameters
+    ----------
+    labels : list
+        A list of feature names.
+    result : list
+        A fingerprint.
+    atoms : object
+        A single atoms object.
+    """
     if len(result) != len(labels):
         msg = str(len(labels)) + '/' + str(len(result)) + \
             ' labels/fingerprint mismatch.'
         if 'id' in atoms.info:
-            msg += ' id: ' + str(atoms.info['id'])
+            msg += ' database id: ' + str(atoms.info['id'])
         raise AssertionError(msg)
 
 
 class AdsorbateFingerprintGenerator(BaseGenerator):
     def __init__(self, **kwargs):
         """Class containing functions for fingerprint generation.
-
```

Comment:
Just FYI, we use the pep8/numpy docstrings style guide which has these additional empty line. Changes to this style can throw errors/warnings when building the docs (http://catlearn.readthedocs.io/en/latest/).